File: VehicleMonitoringSystemMobile/app/src/main/java/com/siroytman/vehiclemonitoringsystemmobile/util/AttachmentUtil.java
package com.siroytman.vehiclemonitoringsystemmobile.util;

import com.siroytman.vehiclemonitoringsystemmobile.api.ApiController;

public class AttachmentUtil {
    public static String getAttachmentUrl(String attachmentName) {
        return ApiController.BACKEND_URL + "/attachment/" + attachmentName;
    }
}

File: VMS_Web/VMS_Web/ClientApp/src/utils/attachmentUtil.ts
import {getBackendServerUrl} from "../api";

export enum MessageTypeConstants {
    TEXT = 'text',
    PHOTO = 'photo',
    // FILE = 'file'
}

export function getAttachmentUrl(attachmentName: string|null): string|null {
    if (!attachmentName) {
        return null;
    }
    return `${getBackendServerUrl()}attachment/${attachmentName}`
}
